pkg/router/route_group.go

```diff
@@ -221,22 +278,32 @@ func (rg *RouteGroup) Write(p []byte) (n int, err error) {
 			return 0, err
 		}
 
+		fmt.Println()
```

Comment:
Did you forget to delete it?

Reply:
Can't even say what was that for lol, but yes, seems so